README.md

```diff
@@ -147,13 +146,14 @@ func main() {
       defer wg.Done()
       // switching host. all options changes are safe to call from goroutines.
       koios.Host(host)(api)
-      res, _ := api.GET(ctx, "/tip")
+      res, _ := api.GET(ctx, "/tip", make(map[string][]string), make(map[string][]string))
```

Comment:
this should be `api.GET(ctx, "/tip", nil, nil)`

Reply:
👍 thx

Reply:
so if you send another commit with that changed it can be merged

Reply:
done